spec/spec_helper.rb

```diff
@@ -27,17 +27,17 @@
 require "rspec/longrun"
 #require "narray/narray"
 
-require "../lib/nmatrix"
-require "../lib/nmatrix/rspec"
+require "./lib/nmatrix"
+require "./lib/nmatrix/rspec"
```

Comment:
@mohawkjohn This change isn't there. I checked . These are previously quoted ones.